src/Http/Session.php

```diff
@@ -226,7 +226,9 @@ public function regenerateId()
 			if (headers_sent($file, $line)) {
 				throw new Nette\InvalidStateException('Cannot regenerate session ID after HTTP headers have been sent' . ($file ? " (output started at $file:$line)." : '.'));
 			}
-			session_regenerate_id(TRUE);
+			if (session_id() !== '') {
+				session_regenerate_id(TRUE);
+			}
 			session_write_close();
```

Comment:
When it is empty on line 229, it should be empty on next line 232 too, isn't it?